pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -115,3 +115,13 @@ type ServiceAffinityArgs struct {
 	// AntiAffinityLabelsPreference are the labels to consider for service anti affinity scoring.
 	AntiAffinityLabelsPreference []string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// VolumeBindingArgs holds arguments used to configure the VolumeBinding plugin.
+type VolumeBindingArgs struct {
+	metav1.TypeMeta
+
+	// BindTimeoutSeconds is the timeout in seconds in volume binding.
+	BindTimeoutSeconds int64
```

Comment:
Wait... I missed that you don't have v1alpha2 counterpart types.....

I guess we can still merge this to leave a follow up for API reviewers

Reply:
Yes, we shouldn't have the versioned types in this PR, we can do it in a separate PR for API review.